src/Modules/Finance/Header/TargetSum/TargetSum.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+
+import styles from '../Form.module.css';
+
+interface ITargetSumProps {
+	value: string | number,

```

Comment:
Since `Header` provides this component with a value of type `string` you can narrow the `value` type to `string`.

Reply:
There's a conditional rendering in line 66 of TargetSum.tsx which requires a 'number' type. I have no solution for that at the moment.

Reply:
The solution would be to narrow the type of the `value` prop, in this case to number. Thus the parent that will instantiate `TargetSum` component will be responsible for passing the `value` prop of correct type (number).